content/docs/install/jes-install/index.md

```diff
@@ -1,13 +0,0 @@
----
-title: Install Job Executor 
```

Comment:
Why do you remove this page. I think it makes sense as a part of the installation guide, especially as a links placeholder as of now

Reply:
Because we said we would not document integrations in the main doc set.  So the list in install/_index.md just links to the JES Integration page.  I admit that I am having some reservations about that practice, especially for JES.  Perhaps we need a more nuanced policy?

Reply:
I fully support this move. We should link to the job-executor README and docs from the main document, and not duplicate things in the Keptn Docs.

Reply:
What we now have is the list of installation tasks links to the JES artifacthub page, which has quickstart installation instructions and a link to full installation instructions in github as well as links to the github pages for intro, features, and architecture.

The alternative was to have a separate file in the docs for installing JES, but that file only contains links to these github pages.  If we restore the file, it should also link to the artifactory page.

Do we agree that we only want to link to the artifactory hub page and let users click into the github pages from there?  Note that neither approach has any actual JES installation information written into the docs, only links.